### PR TITLE
Working SIL for LV controller testing

### PR DESCRIPTION
**Pull request by samparent97**

### CODE REVIEW

File: macformula/pinout/controller.go
Comment (TylerStAmour) [inline macformula/pinout/controller.go:1]: IO Controller is never set here. This also seems to just be a wrapper of iocontrol/iocontrol.go. Should we make iocontrol.go more abstract and define the "controllers" (sg controller, sil controller, etc) in macformula, to then pass the controllers to iocontrol instead? Or is there something else I'm missing here

File: cmd/hilapp/main.go [hunk 1]
```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	"github.com/macformula/hil/macformula/pinout"
```
Comment (TylerStAmour) [inline cmd/hilapp/main.go:7]: Sort

File: macformula/state/frontcontrollerio.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
+package state
+
+import (
+	"context"
+	"github.com/macformula/hil/flow"
+	"github.com/macformula/hil/macformula"
+	"github.com/macformula/hil/macformula/pinout"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+	"time"
```
Comment (TylerStAmour) [inline macformula/state/frontcontrollerio.go:10]: Sort

**Comment by samparent97** [comment]
@TylerStAmour realized today that I made this PR while not having checked that my actual commits were on this branch. Closing this to be replaced by #89

**Pull request closed without merging**